platform/settings/src/org/netbeans/modules/settings/RecognizeInstanceObjects.java

```diff
@@ -93,24 +94,49 @@ protected Lookup lookupFor(Object obj) {
         return null;
     }
     
-    private static final class OverObjects extends ProxyLookup 
-    implements PropertyChangeListener, FileChangeListener {
-        private final String path;
-        
-        public OverObjects(String path) {
-            super(delegates(null, path));
-            this.path = path;
+    private static final class MSL implements PropertyChangeListener {
+        private final PropertyChangeSupport pcs = new PropertyChangeSupport(this);
+        public static final MSL DEFAULT = new MSL();
+        static {
             try {
                 ModuleSystem ms = Main.getModuleSystem(false);
                 if (ms != null) {
                     ModuleManager man = ms.getManager();
-                    man.addPropertyChangeListener(WeakListeners.propertyChange(this, man));
+                    man.addPropertyChangeListener(WeakListeners.propertyChange(DEFAULT, man));
                 } else {
                     LOG.log(Level.WARNING, "Not listening on module system");
```

Comment:
This message was printed out  everytime new instance of `OverObjects` was created in a unit test. Quite annoying. Enough to warning once per test.